hack/quickstart/init-worker.sh

```diff
@@ -34,6 +34,7 @@ function init_worker_node() {
     # Setup kubeconfig
     mkdir -p /etc/kubernetes
     cp ${KUBECONFIG} /etc/kubernetes/kubeconfig
+    cp /home/core/assets/tls/ca.crt /etc/kubernetes/ca.crt
```

Comment:
This won't exist on the worker node - it needs to be copied over (service and kubeconfig are copied here: https://www.pivotaltracker.com/story/show/140894109)

However, I dislike needing to copy another moving part. Right now a worker just needs a kubelet and a valid kubeconfig to be a fully-fledged node.

Just a thought (not sure it's a good idea) - but what about trying to extract CA from the kubeconfig instead (it's already provided in-line)?